### ISSUE TITLE
[Feature Request] - support netstat() plugin and Linux.Network.NetstatEnriched on MacOS

### ISSUE THREAD

**Issue by DYarizadeh** [Velocidex/velociraptor#2936]
Thanks!

**Comment by scudette** [comment]
The netstat plugin appears to be broken right now with this error

```
./velociraptor-v0.7.1-1-darwin-amd64 query -v "SELECT * FROM netstat()"
[INFO] 2024-02-02T03:43:09Z  _    __     __           _                  __
[INFO] 2024-02-02T03:43:09Z | |  / /__  / /___  _____(_)________ _____  / /_____  _____
[INFO] 2024-02-02T03:43:09Z | | / / _ \/ / __ \/ ___/ / ___/ __ `/ __ \/ __/ __ \/ ___/
[INFO] 2024-02-02T03:43:09Z | |/ /  __/ / /_/ / /__/ / /  / /_/ / /_/ / /_/ /_/ / /
[INFO] 2024-02-02T03:43:09Z |___/\___/_/\____/\___/_/_/   \__,_/ .___/\__/\____/_/
[INFO] 2024-02-02T03:43:09Z                                   /_/
[INFO] 2024-02-02T03:43:09Z Digging deeper!                  https://www.velocidex.com
[INFO] 2024-02-02T03:43:09Z This is Velociraptor 0.7.1-1 built on 2023-12-27T12:11:30Z (2d7d6cfa)
[INFO] 2024-02-02T03:43:09Z Env var VELOCIRAPTOR_API_CONFIG is not set
[INFO] 2024-02-02T03:43:09Z No embedded config - you can pack one with the `config repack` command
[INFO] 2024-02-02T03:43:09Z Env var VELOCIRAPTOR_CONFIG is not set
[INFO] 2024-02-02T03:43:09Z Setting empty config
[INFO] 2024-02-02T03:43:09Z Starting Org Manager service.
[INFO] 2024-02-02T03:43:09Z Starting services for Root Org
[INFO] 2024-02-02T03:43:09Z Starting Journal service for Root Org.
[INFO] 2024-02-02T03:43:09Z Starting user manager service for org
[INFO] 2024-02-02T03:43:09Z Starting the notification service for Root Org.
[INFO] 2024-02-02T03:43:09Z Installing Dummy inventory_service. Will download tools to temp directory.
[INFO] 2024-02-02T03:43:09Z Starting repository manager for Root Org
[INFO] 2024-02-02T03:43:09Z Loaded 391 built in artifacts in 271.346202ms
[INFO] 2024-02-02T03:43:09Z netstat: gatherTCP: unexpected end of buffer after 260 records, 24 bytes left
[INFO] 2024-02-02T03:43:09Z netstat: gatherUDP: unexpected end of buffer after 273 record, 24 bytes left
[][INFO] 2024-02-02T03:43:09Z Completed query in 3.591006ms
[DEBUG] 2024-02-02T03:43:09Z Query Stats: {"RowsScanned":0,"PluginsCalled":1,"FunctionsCalled":0,"ProtocolSearch":0,"ScopeCopy":1}
[INFO] 2024-02-02T03:43:09Z Exiting notification service for Root Org!
```

**Comment by scudette** [comment]
Fixed by #3312
